src/routes/(app)/docs/[lang]/[filename]/+server.ts

```diff
@@ -0,0 +1,15 @@
+import { error } from '@sveltejs/kit';
+import fs from 'fs';
+import path from 'path';
+
+export async function GET({ params }) {
+	const { lang, filename } = params;
+	const filePath = path.resolve('docs', lang, filename);
+
+	if (!fs.existsSync(filePath)) {
+		throw error(404, 'File not found');
+	}
+
+	const content = fs.readFileSync(filePath, 'utf-8');
+	return new Response(content, { status: 200, headers: { 'Content-Type': 'text/plain' } });
+}
```

Comment:
Use `rootDir` (like in `geoip.ts`) to be sure to load from the correct path

Also add ̀`docs` here https://github.com/be-BOP-io-SA/be-BOP/blob/941fbbd5bf34b503c1925323aa20907b4c13b3a1/Dockerfile#L30 to the copied files for docker instealls

also, `lang` and `filename` should be validated to not contain any special characters.

In fact,  `lang` should be validated with `z.enum()` against the folders inside of `docs` , and `filename` should then be validated against the `*.md` files in that folder.

To avoid somebody using `../` or the escaped version (`%2F`) to "hack" the site and get the env, eg somebody calling `http://localhost:5172/docs/en/..%2F..%2F.env` (not sure if it works) - even if it doesn't work, we should be very careful and prevent it